radix-engine-interface/src/types/blueprint.rs

```diff
@@ -47,7 +53,14 @@ impl ObjectInfo {
     }
 
     pub fn get_features(&self) -> BTreeSet<String> {
-        self.features.clone()
+        self.blueprint_info.features.clone()
+    }
+
+    pub fn try_get_outer_object(&self) -> Option<GlobalAddress> {
```

Comment:
Super minor: we may just use `get_outer_object`, or update all "optional methods" to have a `try_` prefix, for consistency.